hudi-common/src/main/java/org/apache/hudi/common/model/HoodieLogFile.java

```diff
@@ -147,6 +148,10 @@ public String getFileExtension() {
     return fileExtension;
   }
 
+  public boolean isCDC() {
```

Comment:
idk. I like it here because I think future hudi contributors will be more likely to find it instead of creating duplicate code. Why do you think it should be in FSUtils?